pep/app/utils/csv.ts

```diff
@@ -0,0 +1,11 @@
+import { QueryParamsObj } from '@gavant/ember-pagination/utils/query-params';
+import { urlForDocumentQuery } from 'pep/pods/document/adapter';
+import { serializeQueryParams } from 'pep/utils/url';
+import DS from 'ember-data';
+
+export const csvUrl = (store: DS.Store, queryParams: QueryParamsObj) => {
+    delete queryParams.limit;
+    queryParams.download = true;
+    const url = urlForDocumentQuery(store, { queryType: queryParams.queryType });
+    return `${url}?${serializeQueryParams(queryParams)}`;
+};
```

Comment:
@bakerac4  i probably would throw this in the existing `utils/url.ts` util file, since its probably not likely we'll have too many or any more utils relating to csvs in this project.

Interesting use of the document query adapter to avoid duplicating all that logic. I'd live with it, but I dont _love_ that we have to pass in the `store` context to use this, makes the util a little less portable/standalone. I probably would've just extracted all the url logic from the document adapter's `urlForQuery` method itself, made that a generic util w/o needing the `Store` directly, and then use it both the original adapter, and here. But we can do that in a follow up PR if you want.